CHANGELOG.md

```diff
@@ -5,6 +5,29 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0),
 and this project uses `yyyy.rr[.pp]`, where `yyyy` is the year a patch is released,
 `rr` is a sequential release number (starting from `01`), and an optional two-digit
 sequential patch number (starting from `01`).
+
+## [2022.02] - 2022-04-29
+### Known Issues
+- If using GCC 10 or higher as well as MPICH, compilation errors will occur unless `-fallow-argument-mismatch` is included in the Fortran compiler flags
+- GCC 11.1.0 is unsupported due to compilation issues with select type.
```

Comment:
Should there be a comment here that other versions of GCC 11 are supported?

Reply:
yeah its added now